test/sass/typograpy.spec.scss

```diff
@@ -0,0 +1,164 @@
+@use "sass:math";
+@use "../../src/shared-styles/baseline";
+@use "../../src/shared-styles/font";
+@use "../../src/shared-styles/scale";
+@use "../../src/shared-styles/typography";
+
+@function _calcExpectedFontPx($scale-point) {
```

Comment:
Allows the tests to pass when the base font size and / or modular scale ratio is changed, as long as the correct modular scale point is used to scale the font size